tests/ffi/agent-flood/test.sh

```diff
@@ -64,15 +64,43 @@ run_test_containers(){
     done
 }
 
+# Extract the Network mode from qm quadlet file
+get_qm_network_mode(){
+    qm_config_file="/usr/share/containers/systemd/qm.container"
+
+    # Check if the configuration file exists
+    if [ ! -f "$qm_config_file" ]; then
+        echo "Configuration file not found: $qm_config_file"
+        exit 1
+    fi
+
+    # Extract the Network using awk
+    qm_config_file="/usr/share/containers/systemd/qm.container"
+    qm_network_mode=$(awk -F'=' '/Network=private/ { print $2 }' "$qm_config_file")
```

Comment:
This part is problematic
AFAIK all qm.service unit files should be updated overloaded with the following drop-in qm unit file
here `/etc/containers/systemd/qm.container.d `

Please refer this 
https://github.com/containers/qm/blob/main/tests/ffi/common/prepare.sh#L29

I think we should consult @alexlarsson

Reply:
Maybe it would be fine to use `/run/systemd/generator/qm.service` instead, since by then any drop-in setting would already have been processed?

Reply:
ok this one is not so clear for me.
Do you want to return the Network configuration? 

I would take it from the status of the service 

`systemctl show -p ExecStart qm  | cut -c11- `

Reply:
yes, make more sense

Reply:
changed to `/run/systemd/generator/qm.service`